media/css/115release.css

```diff
@@ -0,0 +1,1178 @@
+  --color-ink-90: #1a1838;
+
+  --color-white: #ffffff;
+  --color-black: #000000;
+
+  --bg: #31293e;
+  --txt: var(--color-gray-05);
+
+  --max-width: 1280px;
+  --slider: 50%;
+
+  --icon-glow: var(--color-gray-50);
+  --info-header: var(--color-gray-20);
+  --info-txt: var(--color-gray-10);
+
+  --button-background-color: var(--color-gray-20);
+  --button-hover-text-color: var(--color-gray-70);
+  --button-hover-background-color: var(--color-gray-10);
+  --button-active-background-color: var(--color-gray-30);
+  --button-border-color: var(--color-gray-40);
+  --button-border-radius: 6px;
+  --button-border-size: 2px;
+  --button-text-color: var(--color-gray-70);
+  --button-margin: 9px;
+  --button-padding: 9px;
+  --focus-outline-color: var(--color-blue-50);
+  --focus-outline-offset: 4px;
+  --focus-outline: 4px solid var(--focus-outline-color);
+  --icon-size: 32px;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-size: 1rem;
+  background-color: var(--bg);
+  color: var(--txt);
+}
+
+header {
+  --max-width: 70ch;
+  background-image: url("/media/img/thunderbird/backgrounds/blob.png");
+  background-size: min(150%, 150vh);
+  background-position: bottom;
+  background-repeat: no-repeat;
+  padding-inline: 16px;
+  padding-block: 100px 500px;
+}
+
+header h1 {
+  font-weight: 300;
+  font-size: 2.5rem;
+  text-shadow: 0 2px 6px var(--color-ink-90);
+}
+
+.tb-title {
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 4rem;
+}
+
+.version,
+.alive {
+  color: transparent;
+  font-weight: 900;
+  line-height: 0;
+  text-shadow: none;
+}
+
+.version {
+  font-size: 10rem;
+  background: linear-gradient(30deg, transparent 25%, var(--color-purple-20));
+  -webkit-background-clip: text;
+  background-clip: text;
+  margin-inline-start: -10rem;
+  margin-block-start: -12rem;
+  vertical-align: super;
+}
+
+.supernova {
+  filter: drop-shadow(0 2px 6px var(--color-ink-90));
+}
+
+.whats-new-section *,
+.container * {
+  box-sizing: border-box;
+}
+
+.whats-new-section {
+  display: grid;
+  place-items: center;
+  min-height: 30rem;
+  gap: 24px;
+  margin-block-end: 128px;
+}
+
+.whats-new-section:first-of-type {
+  margin-block-start: -350px;
+}
+
+img {
+  max-width: 100%;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  max-width: var(--max-width);
+  margin: 0 auto;
+}
+
+.split {
+  flex-direction: row;
+}
+
+.split.reverse {
+  flex-direction: row-reverse;
+}
+
+.split > .info {
+  flex-basis: calc(66% - 12px);
+}
+
+.split > .image {
+  flex-basis: calc(33% - 12px);
+}
+
+.info {
+  padding: 64px 32px;
+}
+
+.slider-box {
+  background-image: url("/media/img/thunderbird/whatsnew/115/new-old-fade.png");
+  background-size: auto 95%;
+  background-position: center;
+  background-repeat: no-repeat;
+  aspect-ratio: 16 / 10;
+  width: 100%;
+  position: relative;
+  isolation: isolate;
+  opacity: 0.75;
+}
+
+.slider-box:hover,
+.slider-box:focus-within {
+  opacity: 1;
+}
```

Comment:
Let's remove all of these.
No need to have opacity on the primary screenshot, it kinda looks bad and it doesn't add anything to the interaction